project.clj

```diff
@@ -7,22 +7,23 @@
                  [cljs-web3 "0.19.0-0-8"]
                  [cljsjs/filesaverjs "1.3.3-0"]
                  [cljsjs/prop-types "15.6.0-0"]
-                 [cljsjs/react "15.6.1-2"]
-                 [cljsjs/react-datepicker "0.55.0-0"]
-                 [cljsjs/react-dom "15.6.1-2"]
-                 [cljsjs/react-infinite "0.12.1-0"]
+                 [cljsjs/react "16.4.1-0"]
+                 [cljsjs/react-datepicker "1.5.0-0"]
+                 [cljsjs/react-dom "16.4.1-0"]
+                 [cljsjs/react-infinite "0.13.0-0"]
                  [cljsjs/react-meta-tags "0.3.0-1"]
                  [day8.re-frame/async-flow-fx "0.0.8"]
                  [day8.re-frame/forward-events-fx "0.0.5"]
-                 ;[honeysql "0.9.1"]
-                 ;; Until PR https://github.com/jkk/honeysql/pull/195 is merged
-                 [madvas/honeysql "0.9.1"]
+                 [honeysql "0.9.3"]
                  [medley "1.0.0"]
                  [org.clojure/clojurescript "1.9.946"]
                  [org.clojure/core.match "0.3.0-alpha4"]
                  [print-foo-cljs "2.0.3"]
-                 [re-frame "0.9.4"]
-                 [re-frisk "0.5.3"]
+                 ;; TODO: On vbump, remove exclusion when new re-frame bumps reagent to 0.8.1+
+                 [re-frame "0.10.5" :exclusions [reagent]]
```

Comment:
why not exclude it globally in `:exclusions` ?

Reply:
I assumed it would remove the new reagent dependency that I added, I'll try putting it in the global exclusions and report back.